gallery/src/components/NxDrawer/NxDrawerPage.tsx

```diff
@@ -0,0 +1,235 @@
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+import React from 'react';
+import {
+  NxTable,
+  NxCode,
+  NxP,
+  NxTextLink,
+  NxH3,
+  NxTile
+} from '@sonatype/react-shared-components';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import NxDrawerExample from './NxDrawerExample';
+import NxDrawerWithSubtitleOrParagraphExample from './NxDrawerWithSubtitleOrParagraphExample';
+import NxDrawerNarrowExample from './NxDrawerNarrowExample';
+import NxDrawerWithFooterExample from './NxDrawerWithFooterExample';
+import NxDrawerEscExample from './NxDrawerEscExample';
+
+const NxDrawerSourceCode = require('./NxDrawerExample?raw');
+const NxDrawerWithGlobalHeaderSourceCode = require('./NxDrawerWithGlobalHeaderExample?raw');
+const NxDrawerWithSubtitleOrParagraphSourceCode = require('./NxDrawerWithSubtitleOrParagraphExample?raw');
+const NxDrawerNarrowSourceCode = require('./NxDrawerNarrowExample?raw');
+const NxDrawerWithFooterSourceCode = require('./NxDrawerWithFooterExample?raw');
+const NxDrawerEscSourceCode = require('./NxDrawerEscExample?raw');
+
+export default function NxDrawerPage() {
+  return (
+    <>
+      <GalleryDescriptionTile>
+        <NxP>
+          The <NxCode>NxDrawer</NxCode> component implements a dialog sidebar that is fixed to the
+          right side of the viewport. Inside it consists of three parts: the header (built-in),{' '}
+          <NxCode>NxDrawer.Content</NxCode> (required), <NxCode>NxDrawer.Footer</NxCode> (optional).
+        </NxP>
+        <NxTile.Subsection>
+          <NxH3>Props</NxH3>
+          <NxTable>
+            <NxTable.Head>
+              <NxTable.Row>
+                <NxTable.Cell>Prop</NxTable.Cell>
+                <NxTable.Cell>Type</NxTable.Cell>
+                <NxTable.Cell>Required</NxTable.Cell>
+                <NxTable.Cell>Default</NxTable.Cell>
+                <NxTable.Cell>Details</NxTable.Cell>
+              </NxTable.Row>
+            </NxTable.Head>
+            <NxTable.Body>
+              <NxTable.Row>
+                <NxTable.Cell>className</NxTable.Cell>
+                <NxTable.Cell>string</NxTable.Cell>
+                <NxTable.Cell>No</NxTable.Cell>
+                <NxTable.Cell></NxTable.Cell>
+                <NxTable.Cell>
+                  Any <NxCode>className</NxCode> attributes passed in on
+                  the <NxCode>NxDrawer</NxCode> will be added to
+                  the dialog element.
+                </NxTable.Cell>
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>onCancel</NxTable.Cell>
+                <NxTable.Cell>Function ((Event) =&gt; void)</NxTable.Cell>
+                <NxTable.Cell>Yes</NxTable.Cell>
+                <NxTable.Cell></NxTable.Cell>
+                <NxTable.Cell>
+                  The callback function passed into <NxCode>NxDrawer</NxCode> that gets called when the user
```

Comment:
Perhaps change this to "NxDrawer's callback function. It gets called when the user closes the dialog"?

Reply:
> Perhaps change this to "NxDrawer's callback function. It gets called when the user closes the dialog"?

Fixed here: 139ae1d5bb5f3141dd1154501a7dff94e2a046d3